### ISSUE TITLE
JSON config files don't work

### ISSUE THREAD

**Issue by PaulMansour** [the-carlisle-group/Acre-Desktop#194]
I'm using Acre  5.0.13+177 .

Are JSON config files supported in this version? Acre will open a project, but it appears to ignore at least the ProjectSpace parameter, setting it to empty, and defaulting to the name of the project under the current namespace.

**Comment by PhilLast** [comment]
I guess you are in Dyalog 17.0 or 17.1.

`7159⌶` and `7160⌶` are no longer supported. Acre's change to use `⎕JSON` if the current Dyalog version exceeds 16.0 was made not long before build 239 which was only a week ago.

Prior to that JSON was supported only for Dyalog 14.1 to 16.0.

Latest builds all support JSON from Dyalog 14.1 on.

But beware:
At the same time the API namespace changed from `⎕SE.acreAPI` to `⎕SE.acre` and all commands and api-fns were consolidated and re-spelled to be identical in PascalCase.

**Comment by PhilLast** [comment]
Another concurrent change now throws an error in case of a problem reading the config file rather than carrying on as if it didnt exist - which is what you experienced.

**Comment by PaulMansour** [comment]
> Latest builds all support JSON from Dyalog 14.1 on.

What's a build? Do you mean the latest commits? The latest version is still  5.0.13 +177

**Comment by PhilLast** [comment]
More frequent than commits. I have to rebuild the system so my changes appear ⎕SE. I increase the build number each time. Commit when a bunch of changes seem to hang together with breaking. 

Release 5.0.13 was build 177. 

I was also increasing the version number until Kai told me I should only do that for a release so the version number of latest build and commit is arbitrarily 5.2.0 as it has been since build 187 on May 20. 

Latest build is 246 from first thing this morning. Latest commit was 244 yesterday.

Currently awaiting Kai's go-ahead to make a new release - that will be 6.0.0 - when Fire has caught up with relevant changes to API.

**Comment by PhilLast** [comment]
**without breaking**

**Comment by PaulMansour** [comment]
Ok, so from my perspective, a build is just commit to the master, as I have no access to intermediate builds.

Why are Acre releases dependent on Fire? I understand that Fire is dependent on Acre, but so is AcreTools, and you don't wait for me to put out a release

In the mean time,  I assume there is an easy way for me to do my own build off a clone of Acre-Desktop? I think I only need a workspace and a .dyalog file.   Is that documented somewhere?

**Comment by PhilLast** [comment]
Thought it was but can't find it right now.

**_)LOAD build.dws_** from the acre project folder with the earliest Dyalog version you're likely to want to use with acre. It will Prompt you with something like:
```
?     )LOAD "C:\Users\Paul\AppData\Local\acre\acre14.0\acre.dws"
```
Ignore that and instead copy the folder **_C:/Users/Paul/AppData/Local/acre/acre14.0/_** to wherever you want your installation and **_)LOAD acre.dws_** from there in whichever Dyalog versions you want.

**Comment by PhilLast** [comment]
Released Acre 6.0.0+248 at 2019-08-22 21:27 UTC

If you haven't followed intructions above you can get the release without the project by downloading the acre14.0.zip.

And in answer to your specific question above:
>Why are Acre releases dependent on Fire? I understand that Fire is dependent on Acre, but so is AcreTools, and you don't wait for me to put out a release

I could if you asked me. Kai was aware of the changes I was making because he was writing the tests for them. So he asked me to wait while he altered Fire and put out a new release of that that wouldn't break because of them.

**Comment by PhilLast** [comment]
Latest push:
 Acre 8.2.0+324 2020-07-01 11:57

Acre in Dyalog v14.0  will now read a JSON acre.config file.
Now all versions covered re: reading config file
